### PR TITLE
MODLD-424: Derive marc 111 from Creator of Work (Meeting) Resource de…

### PR DESCRIPTION
**Pull request by askhat-abishev**
…scription

### CODE REVIEW

File: src/test/java/org/folio/marc4ld/mapper/field111/BibframeToMarc111IT.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+@EnableConfigurationProperties
+@SpringBootTest(classes = SpringTestConfig.class)
+class BibframeToMarc111IT {
+
+  @Autowired
+  private Bibframe2MarcMapperImpl bibframe2MarcMapper;
+
+  @Test
+  void shouldMapField110() {
```
Comment (AndreiBordak) [inline src/test/java/org/folio/marc4ld/mapper/field111/BibframeToMarc111IT.java:46]: looks like it should be 111

File: src/test/java/org/folio/marc4ld/mapper/field111/BibframeToMarc111IT.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    //given
+    var resource = createResourceWithWorkWith111();
+    var expectedMarc = loadResourceAsString("fields/111/marc_111.jsonl");
+
+    //when
+    var result = bibframe2MarcMapper.toMarcJson(resource);
+
+    //then
+    AssertionsForClassTypes.assertThat(result)
+      .isEqualTo(expectedMarc);
```
Comment (AndreiBordak) [inline src/test/java/org/folio/marc4ld/mapper/field111/BibframeToMarc111IT.java:56]: minor: can be single line statement

**Review by AndreiBordak — APPROVED** [review]

**Review by pkjacob — APPROVED** [review]

File: src/test/java/org/folio/marc4ld/mapper/field111/BibframeToMarc111IT.java (hunk 1 above)
Comment (askhat-abishev) [reply src/test/java/org/folio/marc4ld/mapper/field111/BibframeToMarc111IT.java]: renamed

File: src/test/java/org/folio/marc4ld/mapper/field111/BibframeToMarc111IT.java (hunk 2 above)
Comment (askhat-abishev) [reply src/test/java/org/folio/marc4ld/mapper/field111/BibframeToMarc111IT.java:56]: left as is to align with other similar tests

**Pull request merged**